### ISSUE TITLE
[Accessibility] A11y_Manage.IIS.Net_Web Server_Files_Screen Reader: Narrator is not announcing the sorting action that can be done on table headers

### ISSUE THREAD

**Issue by InduPriya1805** [Microsoft/IIS.WebManager#189]
GitHubTags: #A11y_Manage.IIS.Net; #A11yMAS; #A11yTCS; #C+E-H; #DesktopWeb; #Win10; #.NETWebsites; #Edge; #ScreenReader; #Narrator; #Chrome; #NVDA; #MAS1.3.1

**Environment Details:**
URL: https://manage.iis.net/webserver/files#inetpub/
Browser Details: Microsoft Edge 42.17134.1.0, Google Chrome 68.0.3440.106
Screen Reader: Narrator, NVDA
MAS ID Violated: MAS 1.3.1

**Repro Steps:**
1. Download and install "Microsoft IIS Administration" application.
2. Enable Internet information Services.
3. Open "Microsoft IIS" application using the URL: https://manage.iis.net/
4. Tab till Web Server and press enter
5. Tab till Files and press enter
6. Turn on narrator in scan mode
7. Tab till name and verify Narrator is announcing the sorting action that can be done on table headers or not

**Actual Result:**
Narrator is not announcing the sorting action that can be done on table headers

**Expected Result:**
Narrator should announce the sorting action that can be done on table headers

**Note:**
This type of issue is observed across all the pages in the application which has table headers with sorting option
In NVDA, it is reading as clickable but it is not specifying the type of action that we can done (it should also read the action ex: sorting)

**User Impact:**
If Narrator is not announcing the sorting action that can be done on table headers screen reader users doesnot know about the sorting option present over the table headers

Attachments:
 
[Byg189_files_narrator not specifying the action.pptx](https://github.com/Microsoft/IIS.WebManager/files/2335570/Byg189_files_narrator.not.specifying.the.action.pptx)

**Comment by jhkimnew** [comment]
Fixed

**Comment by prabhudevu** [comment]
**GitHubTags:**#A11ySev2;

**Comment by fsteffi** [comment]
#Closed;
Manage.iis.net is deprecated. The "Manage IIS" button on the [www.iis.net](http://www.iis.net/) home page will be removed.
